ui/src/containers/CreateVolume.js

```diff
@@ -435,13 +453,64 @@ const CreateVolume = (props) => {
                   }
                 }
               };
+              // TODO: Apply this function as `labelName` field validation function, for the moment we only have one error message.
```

Comment:
Is this todo still valid ? I see various error message in the implementation.

Reply:
OK. So in the end I switched to `Field-level Validation` instead of `Yup` schema which will display the error message separately for the `label prefix` and `labe name value`.
Please see: https://github.com/scality/metalk8s/pull/3338/commits/9917a17a96db87f73451d6d13ff38e6b7db8de77